ddtrace/contrib/dbapi/__init__.py

```diff
@@ -47,6 +50,12 @@ def _trace_method(self, method, name, resource, extra_tags, *args, **kwargs):
             s.set_tags(pin.tags)
             s.set_tags(extra_tags)
 
+            # set analytics sample rate if enabled
+            s.set_tag(
+                ANALYTICS_SAMPLE_RATE_KEY,
```

Comment:
We want to make sure this is only enabled on execute/query spans and not `fetch*`, or `rollback`, etc.

Reply:
If someone enabled the `fetch*` methods, their bill will blow up like crazy